roles/cs.supervisor/tasks/main.yml

```diff
@@ -11,11 +11,34 @@
   when: supervisor_programs | length > 0
   notify: Reload supervisord
 
-- name: Force a restart if configuration has changed
-  meta: flush_handlers
+- name: Create systemd service file for supervisord
+  copy:
```

Comment:
This completely replaces `/usr/lib/systemd/system/supervisord.service`, preferred way to alter distribution provided service files is by using overrides (like that: https://github.com/mageops/ansible-infrastructure/blob/rocky-master/roles/cs.varnish/tasks/002-configure.yml#L47-L49)
In that way we can only alter what we want without cutting ourselves from distribution provided version.